typings/device-overall-status.d.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Extracted from the enum defined in `device-ts.ts`.
+ */
+export enum DeviceOverallStatus {
```

Comment:
Why is there a separate .d.ts file for this? The type is already available from the .ts file so will have an auto generated .d.ts for it

Reply:
@Page- we chatted with Thodoris about it, and you cannot import a `.ts` file in `d.ts` since the `.ts` file gets compiled away and the import will fail.

Reply:
Ok, I see why you need it now, I'd say it makes sense to rename device-ts.ts to device-overall-status.ts then to make it a little clearer that this is just the .d.ts for it (which we need to commit for the sake of the device.d.ts